ios/NativeSigner/Extensions/Models/Encryption+RawRepresentable.swift

```diff
@@ -7,7 +7,6 @@
 
 import Foundation
 
-
```

Comment:
This is just a SwiftFormat correction from last PR where I didn't trigger build before commiting 🙏🏻